ansible/library/pulp_facts.py

```diff
@@ -50,6 +50,10 @@
 # Remove any duplicates
 rpm_dependency_list = list(set(rpm_dependency_list))
 
+# Replace python-lxml with python2-lxml
```

Comment:
It would be helpful to include a short explanation of why this is necessary, or perhaps just a link to the email thread.